### PR TITLE
Extract colours from gs-container, use separate class

### PR DESCRIPTION
**Pull request by davidrapson**
`.gs-container` classes should ideally not have any non-layout styles on them, this was causing some inflexibility in how we do layouts so it's worth sorting.

This PR adds a separate container to determine colour and leaves the `.gs-container` class for constraining layout only. I've also updated all wrapping container to use `<main>`.

*(Also removes an unused view)*

@TesterSpike Would you mind giving this one a once through. If i've missed a page then you will see a grey background for the whole page instead of a white container.

@afiore @tudorraul

### CODE REVIEW

File: app/views/checkout/alldone.scala.html [hunk 1]
```diff
@@ -1,17 +0,0 @@
-@import configuration.Config.Identity.webAppProfileUrl
```
Comment (tudorraul) [inline app/views/checkout/alldone.scala.html]: was this template not used?

Comment (davidrapson) [inline app/views/checkout/alldone.scala.html]: Not anymore, no.

**Comment by tudorraul** [comment]
looked to be a bigger change initially, but I think that was due to the diffing algorithm.
:+1: LGTM, bar the comment

**Comment by TesterSpike** [comment]
:+1:

**Pull request merged**